configure.ac

```diff
@@ -42,6 +43,7 @@ AM_PROG_CC_C_O
 AX_ADD_FORTIFY_SOURCE
 AX_CXX_COMPILE_STDCXX([11], [noext], [mandatory])
 PKG_PROG_PKG_CONFIG
+AX_PTHREAD
```

Comment:
This macro is used immediately below already, so if there's something to fix it's probably in the existing usage.